### ISSUE TITLE
[EPIC] Product page - Options tab 9/9

### ISSUE THREAD

**Issue by marionf** [PrestaShop/PrestaShop#14777]
### -As a merchant I want to be able to manage the options of my products

https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/catalog/catalog-products-add-edit.md#options-tab-epic

- [x] #19679
- [x] #19680
- [x] #19681
- [x] #19682
- [x] #19683

### [- As a merchant I want to know if there are errors when I complete a field in the product page](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/catalog/catalog-products-add-edit.md#options-tab-)

---

### - Issues in **options tab** sorted by priority high to low :

**Nice to Have**
- [x] #9580
- [x] #15553
- [x] #28430

**Feature**
- [ ] #19179
- [ ] #17362 - **[PR]**
- [x] #14135
- [x] #11795
- [ ] #12095 - **`TBS`**

## Fixed
- [x] #27659
- [x] #18919
- [x] #21633
- [x] #22123
- [x] #9790
- [x] #16272

**Comment by marionf** [comment]
### - Issues in **options tab** sorted by priority high to low :

**Nice to Have**
- [ ] #9580
- [ ] #15553

**Improvement**
- [ ] https://github.com/PrestaShop/PrestaShop/issues/19179 - Module
- [ ] #17362
- [ ] #14135
- [ ] #11795

## Fixed
- [x] #18919 - Multistore
- [x] #21633 [BO][Products][Product customization] checkbox replace switch only after saving
- [x] #9790
- [x] #16272
